.github/workflows/release.yml

```diff
@@ -42,9 +42,47 @@ jobs:
           channel: "stable"
       - name: Run CI
         run: bash ./scripts/run_ci_tasks.sh -i
+  upload-docs:
+    runs-on: macos-latest
```

Comment:
do we need this on macos-latest?

Reply:
I chose mac to have the same environment for the test but we can use ubuntu-latest also.